### ISSUE TITLE
RemovedInDjango19Warning: django.utils.importlib

### ISSUE THREAD

**Issue by ntwalibas** [django-tastypie/django-tastypie#1307]
So I downloaded the latest version of tastypie by running `pip install -e git+https://github.com/django-tastypie/django-tastypie#egg=TastyPie`. But opening a url got me an `ImportError` with message `cannot import name resources`.
So to dig a little bit, I launched the shell with `python manage.py shell` and tried to import resources like so: `import from tastypie resources`. And then I got this:

```shell
RemovedInDjango19Warning: django.utils.importlib will be removed in Django 1.9.
  from django.utils import datetime_safe, importlib
```
What is error due to? I am using Django 1.8 on Python 2.7.  
Anyway to get tastypie to work on Django 1.8?

**Comment by SeanHayes** [comment]
That's just a warning that the feature will be removed in the next version of Django, your code should work fine right now.

**Comment by ntwalibas** [comment]
Except it's not working. I wrote this right away after failing to get the import working. Or am I importing the resources wrong?
This is how I am doing it:
```python
from tastypie import resources
```
What's the proper of doing it?

**Comment by SeanHayes** [comment]
Warnings shouldn't cause your program to fail.

Were there any other errors? A stack trace?

**Comment by SeanHayes** [comment]
Sorry, I missed part of your question.

run this:
pip uninstall TastyPie
pip uninstall django-tastypie

pip install -e git+https://github.com/django-tastypie/django-tastypie#egg=django-tastypie

**Comment by ntwalibas** [comment]
I can confirm things are working very well here.
I did uninstall the previous version of tastypie so that was not a problem.
Right when I was giving up (I commented everything tastypie related), you reopen the issue with more comments.
So I un-commented and rerun the app to get a stacktrace for you and then it decided to work!
My best guess it that something was cached until after Django detected the change in code, it decided to update its caches.

So to anyone in the future encountering this: force Django to reload caches by making a change to your source code.
That is at least my explanation for what happen.

Any more enlightenment into how this happened is very much appreciated.
